### ISSUE TITLE
Sticky Gibs Texture

### ISSUE THREAD

**Issue by qolpit** [Ramalayha/gibsplat2#13]
I think the sticky gibs are meant to have some sort of fleshy texture but they remain as watermelon chunks
![image](https://user-images.githubusercontent.com/86146343/122635895-b24a2500-d129-11eb-9217-b99b6462f0b7.png)

**Comment by qolpit** [comment]
![image](https://user-images.githubusercontent.com/86146343/122636015-58962a80-d12a-11eb-8960-ed612f85ad68.png)

**Comment by Ramalayha** [comment]
Do you have graphics addons installed? There's an addon called enhanced prop rendering that has caused issues for others before.

**Comment by qolpit** [comment]
No, I only have gibsplat 2, some maps, npcs and weapon mods. Nothing that really affects rendering
